app/models/legacy/loan_question.rb

```diff
@@ -7,6 +7,25 @@ class LoanQuestion < ActiveRecord::Base
 
     remove_method :id, :question
 
+    # Helper method to sanitize local mysql db until we have cleaned up production data.
+    def self.clean_up_legacy_db
+      # mirror new group and order fields for post analysis questions
+      connection.execute("UPDATE LoanQuestions set NewGroup = Grupo where active = 3")
+      connection.execute("UPDATE LoanQuestions set NewOrder = Orden where active = 3")
+
+      # tweak so that cross-linked children will be marked as 'is_active'
+      connection.execute("UPDATE LoanQuestions set Active = 4 where Active = 2 and NewGroup is not null")
+
+      # filter out questions which seemed to be ignored by old system.
+      connection.execute("UPDATE LoanQuestions set Active = 0 where NewOrder = 0")
+      connection.execute("UPDATE LoanQuestions set Active = 0 where NewGroup is null and Type != 'Grupo'")
+      # ignore some random cruft from lastest prod db
+      connection.execute("UPDATE LoanQuestions set Active = 0 where Type = ''")
+
+      # create some test data for the 'inactive' state
+      connection.execute("update LoanQuestions set NewGroup = Grupo where active = 2 and id > 100")
+    end
```

Comment:
I was working on a clone of the db, so I wasn't thinking from the context playing nicely with the existing production system.  I was actually expecting those additional changes to be performed manually on production.  I'll just comment them out from what gets committed.  They're handy though to be able to temporarily uncomment to cleanup recloned production data.